src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -1,41 +1,41 @@
+import { Location } from '@angular/common';
 import { HttpClient, HttpErrorResponse, HttpHeaders, HttpResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { GitHubDetails, GitProviderService, HelperService } from 'ngx-launcher';
 import { AuthenticationService } from 'ngx-login-client';
-import { empty as observableEmpty, Observable, of as observableOf,  throwError as observableThrowError } from 'rxjs';
+import { empty as observableEmpty, Observable, of, of as observableOf, throwError as observableThrowError } from 'rxjs';
```

Comment:
There are two different imports for `of` with different names. Also, for consistency i would prefer if we use actual namespaces for `of`, `empty` and `throwError`. It would be less confusing if its named same all over the application.